examples/lidar_visual/Main.cc

```diff
@@ -219,7 +219,7 @@ LidarVisualPtr createLidar(ScenePtr _scene)
   // LVT_RAY_LINES -> Lines along the lidar sensor to the obstacle
   // LVT_TRIANGLE_STRIPS -> Coloured triangle strips denoting hitting and
   // non-hitting parts of the scan
-  lidar->SetType(LidarVisualType::LVT_RAY_LINES);
+  lidar->SetType(LidarVisualType::LVT_TRIANGLE_STRIPS);
```

Comment:
Is there any reason to change this?

Reply:
I did this as LVT_TRIANGLE_STRIPS was the standard visual before ( before we added types, and the only visual in Gazebo Classic), in order to retain the same in the example.